Gemfile

```diff
@@ -33,6 +33,8 @@ gem 'bootsnap', '>= 1.4.4', require: false
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  # Rspec Rails https://github.com/rspec/rspec-rails/
+  gem 'rspec-rails', '~> 5.0', '>= 5.0.1'  
```

Comment:
Nice , I think we will move this to group :develoment and :test , we have it in test group right now